video/cloud-client/README.md

```diff
@@ -59,3 +59,12 @@ Detect Shots
 java -cp target/video-google-cloud-samples-1.0.0-jar-with-dependencies.jar \
     com.example.video.Detect shots gs://cloudmleap/video/next/gbikes_dinosaur.mp4
 ```
+
+From Windows, you may need to supply your classpath diferently, for example:
+```
+java -cp target\\video-google-cloud-samples-1.0.0-jar-with-dependencies.jar com.example.video.Detect labels gs://demomaker/cat.mp4
+```
+or
+```
+java -cp target\\video-google-cloud-samples-1.0.0-jar-with-dependencies.jar com.example.video.Detect labels-file resources/cat.mp4
```

Comment:
should n't 'resources/cat.mp4' also be 'resources\\cat.mp4' ?
I'm concerned of maintenance / consistency of these comments across our samples : is there a general statement we can make in our samples that paths to local directories are made in Unix style and need to be updated to the backslash for Windows environments ?